.circleci/config.yml

```diff
@@ -19,6 +19,11 @@ commands:
       - run:
           name: Setup gradle.properties
           command: cp gradle.properties-example gradle.properties
+  update-gradle-memory:
+      steps:
+        - run:
+            name: Update memory setting
+            command: sed -i "s/org.gradle.jvmargs=.*/org.gradle.jvmargs=-Xmx1537m -XX:+HeapDumpOnOutOfMemoryError/" gradle.properties
```

Comment:
I was initially testing this with `Xmx1537m` to make sure my current value `Xmx1536m` was changing. `1536m` is a nice round value, but I don't think it matters if we use `1537m`, so I am not creating an extra commit and triggering CircleCI just for 1 mb :)

Reply:
sgtm! :-)